container-search/src/main/java/com/yahoo/search/handler/SearchHandler.java

```diff
@@ -410,7 +413,7 @@ private Result search(String request, Query query, Chain<Searcher> searchChain)
         } catch (ParseException e) {
             ErrorMessage error = ErrorMessage.createIllegalQuery("Could not parse query [" + request + "]: "
                                                                  + Exceptions.toMessageString(e));
-            log.log(LogLevel.DEBUG, () -> error.getDetailedMessage());
```

Comment:
Why are you removing the function indirection in these?

Reply:
That was supposed to be method reference, not method call. Fixed